### PR TITLE
Place holder for the EE 10 Core Profie specification content

### PR DESCRIPTION
**Pull request by starksm64**
Signed-off-by: Scott M Stark <starksm64@gmail.com>

## Specification PR template
When creating a specification project release review, create PRs with the content defined as follows.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of jakarta-wombat-spec-x.y.pdf
 N/A
- [x] Specification HTML in the form of jakarta-wombat-spec-x.y.html
  N/A
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [x] For a Progress Review, that sufficient progress has been made on a Compatible Implementation and TCK, to ensure that the spec is implementable and testable.
  N/A
- [x] For a [Release Review](https://www.eclipse.org/projects/handbook/#release-review), a summary that a Compatible Implementation is complete, passes the TCK, and that the TCK includes sufficient coverage of the specification. The TCK users guide MUST include the instructions to run the compatible implementations used to validate the release.
Instructions MAY be by reference.
   N/A
- [x] The URL of the OSSRH staging repository for the api, javadoc:
   N/A
- [x] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
    N/A
- [x] The URL of the compatibility certification request issue:
    N/A
- [x] Specification JavaDoc in the wombat/x.y/apidocs directory. 
  N/A

### CODE REVIEW

File: coreprofile/_index.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 ---
 title: "Jakarta EE Core Profile"
-weight: -3
-summary: "The Jakarta EE Core Profile defines a profile of the Jakarta EE platform specifically targeted at smaller runtimes.
-"
+weight: -5
```
Comment (ivargrimstad) [inline coreprofile/_index.md:3]: -3

File: coreprofile/_index.md [hunk 2]
```diff
@@ -1,9 +1,8 @@
...
+summary: "The Jakarta EE Core Profile defines a profile of the Jakarta EE Platform targetting cloud native microservice applications."
```
Comment (ivargrimstad) [inline coreprofile/_index.md:4]: I am good with this rewording, but if I recall correctly there was some resistance to  the use of "cloud native"? But I may be mistaken...

File: coreprofile/_index.md [hunk 3]
```diff
@@ -1,9 +1,8 @@
...
 #<!--.................0123456789.123456789.123456789.123456789.123456789.123456789-->
-summary_sixty_char: "Core profile of platform for smaller runtimes"
+summary_sixty_char: "Core profile of platform for Jakarta EE cloud native apps"
```
Comment (ivargrimstad) [inline coreprofile/_index.md:6]: Same as above. The "smaller runtimes" was sort of a compromise?

File: coreprofile/10/_index.md [hunk 4]
```diff
@@ -1,37 +1,66 @@
...
+| Werner Keil                                    | Committer Members   |   +1   |
+| Jun Qian                                       | Enterprise Members  |   +1   |
+|                                                | Total               | **10** |
+
+The ballot was run in the [jakarta.ee-spec mailing list](https://www.eclipse.org/lists/jakarta.ee-spec/msg01679.html).
+
+## Release Review
+<!--
+TBD
 The Specification Committee Ballot concluded successfully on 2021-05-19 with the following results.
```
Comment (ivargrimstad) [inline coreprofile/10/_index.md:49]: This is the result of the "Creation and Plan Review". If the plan part is replaced by the ballot above, the creation review should at least be listed.

File: coreprofile/10/_index.md [hunk 5]
```diff
@@ -1,37 +1,66 @@
...
+* [Jakarta EE 10 Schedule](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10#jakarta-ee-10-schedule)
 
+# Compatible Implementations
+* [Jakarta EE 10 Compatible Implementations](https://jakarta.ee/compatibility/#tab-10)
 
 # Ballots
 
 ## Plan Review
 
+The Specification Committee Ballot concluded successfully on 2021-08-19 with the following results.
```
Comment (ivargrimstad) [inline coreprofile/10/_index.md:28]: Is this actually correct? The "Creation and Plan Review" was conducted in May. It was not a part of the ballot for Platform and Web Profile.

**Review by ivargrimstad — CHANGES_REQUESTED** [review]
Some comments

File: coreprofile/10/_index.md (hunk 4 above)
Comment (starksm64) [reply coreprofile/10/_index.md:49]: That section is for a future release and is commented out. The creation/plan review is what is referecned as the https://www.eclipse.org/lists/jakarta.ee-spec/msg01679.html vote, so I corrected the date and updated the title.

File: coreprofile/10/_index.md (hunk 5 above)
Comment (starksm64) [reply coreprofile/10/_index.md]: I have corrected that.

File: coreprofile/_index.md (hunk 1 above)
Comment (starksm64) [reply coreprofile/_index.md]: Changed

File: coreprofile/_index.md (hunk 2 above)
Comment (starksm64) [reply coreprofile/_index.md]: Changed

File: coreprofile/_index.md (hunk 3 above)
Comment (starksm64) [reply coreprofile/_index.md]: Changed

**Review by ivargrimstad — APPROVED** [review]

**Comment by starksm64** [comment]
Out of date

**Pull request closed without merging**